src/components/WorldListPage/WorldListPage.tsx

```diff
@@ -106,22 +118,20 @@ const WorldListPage: React.FC<Props> = props => {
   const paginate = useCallback((): ENS[] => {
     const list = tab === TabType.DCL ? ensList : externalNames
 
-    return list
-      .sort((a: ENS, b: ENS) => {
-        switch (sortBy) {
-          case SortBy.ASC: {
-            return a.subdomain.toLowerCase() > b.subdomain.toLowerCase() ? 1 : -1
-          }
-          case SortBy.DESC: {
-            return a.subdomain.toLowerCase() < b.subdomain.toLowerCase() ? 1 : -1
-          }
-          default: {
-            return 0
-          }
+    return list.sort((a: ENS, b: ENS) => {
```

Comment:
If we sort the domains here lexicographically, the sorting might break the pagination, as when new domains are fetched, they could be lexicographically before, after or in the same the page the user is seeing. The sorting should be done at the query level for it to work. We can also remove the sorting completely.